tg/models.py

```diff
@@ -608,12 +608,27 @@ def is_online(self, user_id: int) -> bool:
             return True
         return False
 
+    def get_user_full_info(self, user_id: int) -> Dict[str, Any]:
+        user = self.get_user(user_id)
+        if not user:
+            return user
+        if user.get("full_info"):
+            return user["full_info"]
+
+        result = self.tg.get_user_full_info(user_id)
+        result.wait()
```

Comment:
tip: instead of doing pretty ugly `wait()` in caller's code you can and `block=True` to tdlib call. See https://github.com/paul-nameless/tg/blob/50b93eb3087c02af689246ba4816e1ebda31dc6f/tg/tdlib.py#L118